src/runtime/internal/ssr.ts

```diff
@@ -32,16 +32,31 @@ export function spread(args, classes_to_add) {
 	return str;
 }
 
-export const escaped = {
+const ATTR_REGEX = /[&"]/g;
+const CONTENT_REGEX = /[&<]/g;
+
+const escapes = {
 	'"': '&quot;',
-	"'": '&#39;',
 	'&': '&amp;',
-	'<': '&lt;',
-	'>': '&gt;'
+	'<': '&lt'
 };
 
-export function escape(html) {
-	return String(html).replace(/["'&<>]/g, match => escaped[match]);
+export function escape(html: string, attr: 0 | 1 = 0) {
+	if (typeof html !== 'string') return html;
+
+	const pattern = (attr ? ATTR_REGEX : CONTENT_REGEX);
```

Comment:
probably don't need the parens on this line. though I imagine the ts compiler will take them out anyway